aospy/data_loader.py

```diff
@@ -243,14 +204,21 @@ def _load_data_from_disk(file_set, preprocess_func=lambda ds: ds,
     Returns
     -------
     Dataset
+
     """
     apply_preload_user_commands(file_set)
     func = _preprocess_and_rename_grid_attrs(preprocess_func, grid_attrs,
                                              **kwargs)
-    return xr.open_mfdataset(file_set, preprocess=func, concat_dim=TIME_STR,
-                             decode_times=False, decode_coords=False,
-                             mask_and_scale=True, data_vars=data_vars,
-                             coords=coords)
+    return xr.open_mfdataset(
+        file_set,
+        preprocess=func,
+        combine='by_coords',
```

Comment:
Great, I agree; `'by_coords'` makes the most sense for aospy's use cases.  We want times to be concatenated in order.